portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportTemplate.testcase

```diff
@@ -70,6 +71,7 @@ definition {
 			exportTemplateName = "Export Template Name");
 	}
 
+	@description = "Verify is possible export a template with relaunched process."
```

Comment:
The testcase name is misleading. The test point is that able to relaunch the exported process.

Reply:
@JoyceWang08211 Done!